Marlin/Marlin_main.cpp

```diff
@@ -9496,39 +9496,6 @@ inline void gcode_M502() {
 
 #endif // ADVANCED_PAUSE_FEATURE
 
-#if ENABLED(MK2_MULTIPLEXER)
-
-  inline void select_multiplexed_stepper(const uint8_t e) {
-    stepper.synchronize();
-    disable_e_steppers();
-    WRITE(E_MUX0_PIN, TEST(e, 0) ? HIGH : LOW);
-    WRITE(E_MUX1_PIN, TEST(e, 1) ? HIGH : LOW);
-    WRITE(E_MUX2_PIN, TEST(e, 2) ? HIGH : LOW);
-    safe_delay(100);
-  }
-
-  /**
-   * M702: Unload all extruders
-   */
-  inline void gcode_M702() {
-    for (uint8_t s = 0; s < E_STEPPERS; s++) {
-      select_multiplexed_stepper(e);
-      // TODO: standard unload filament function
-      // MK2 firmware behavior:
-      //  - Make sure temperature is high enough
-      //  - Raise Z to at least 15 to make room
-      //  - Extrude 1cm of filament in 1 second
-      //  - Under 230C quickly purge ~12mm, over 230C purge ~10mm
-      //  - Change E max feedrate to 80, eject the filament from the tube. Sync.
-      //  - Restore E max feedrate to 50
-    }
-    // Go back to the last active extruder
-    select_multiplexed_stepper(active_extruder);
-    disable_e_steppers();
-  }
-
-#endif // MK2_MULTIPLEXER
```

Comment:
The code `M702` is provided in the MK2 branch, and so is included here for consistency. Note that the MK2 multiplexer can't just switch like every other tool, invisibly with no fanfare. The filament **always** has to back out of the extruder and then the new filament **always** has to be inserted.

While it's "okay" to have the stepper ISR manage switching the multiplexer, doing it there adds (almost exclusively wasted) overhead to every single stepper ISR call.